net/modemmanager/Makefile

```diff
@@ -64,6 +64,7 @@ MESON_ARGS += \
 	-Dintrospection=false \
 	-Dman=false \
 	-Dbash_completion=false \
+	-Dbuiltin_plugins=true \
```

Comment:
I don't like the idea of including the plugins in the ModemManager binary. All plugins, whether they are needed or not, have to be loaded into RAM when the ModemManager is started. I think this is currently not the case for plugins that are not needed at runtime.

Reply:
The modemmanager author @aleksander0m wrote that this option works faster. Also, my total package size has become smaller.

Reply:
All plugins whether needed or not we're being loaded at runtime. Including all in the MM binary is much better, if there is no plan to allow third party plugins

Reply:
Up to now, I think there is no plan to support third party plugins in openwrt. So I think this is OK. But I am little surprised, that the ModemManager is realy smaller, if the plugins are all build in! I would not have expect that.